composer.json

```diff
@@ -33,7 +33,7 @@
 		}
 	],
 	"require": {
-		"php": ">5.3.0,~5.5.0"
+		"php": ">5.3.0,~5.6.0"
```

Comment:
Thought about it: This is fine with me, we actually should raise the minimum PHP version to at least 5.4, maybe even 5.5. 

We shouldn't support EOL PHP versions, even if CMS might say otherwise.
http://php.net/supported-versions.php